packages/tokens/package.json

```diff
@@ -25,7 +25,8 @@
   "devDependencies": {
```

Comment:
now `.js` extension without `type:module` won't work as expected.

we need to use `type:module` to "turn on" proper exports resolution ( otherwise commonjs will be always used ).

This has implications on all tooling because once you set `type:module` all imports or file extensions need to be adjusted to follow proper module resolution.

this will fail miserably with `just-scripts` for instance, as it has no support for ESM (haven't check other  tools, jest is probably next on the list, storybook works with tweaks - I setup that for wc-3 branch )